overlord/hookstate/ctlcmd/get.go

```diff
@@ -282,24 +282,42 @@ func (c *getCommand) getInterfaceSetting(context *hookstate.Context, plugOrSlot
 
 	var which string
 	if c.ForcePlugSide || (isPlugSide && !c.ForceSlotSide) {
-		which = "plug-attrs"
+		which = "plug"
 	} else {
-		which = "slot-attrs"
+		which = "slot"
 	}
 
 	st := context.State()
 	st.Lock()
 	defer st.Unlock()
 
-	attributes := make(map[string]interface{})
-	if err = attrsTask.Get(which, &attributes); err != nil {
+	staticAttrs := make(map[string]interface{})
+	dynamicAttrs := make(map[string]interface{})
+	if err = attrsTask.Get(fmt.Sprintf("%s-static", which), &staticAttrs); err != nil {
```

Comment:
Do you have to `make` those? Isn't `Get` going to do that for you? The same question applies to similar code later in this diff.

Reply:
Thanks for spotting, not needed indeed. Fixed.